tests/test.yml

```diff
@@ -169,14 +169,14 @@
     command: >
       oc -n test-non-admin get role osbs-custom-build
     register: cmd_role
-    failed_when: "'No resources found' not in cmd_role.stderr"
+    failed_when: ('No resources found' not in cmd_role.stderr) and ('NotFound' not in cmd_role.stderr)
```

Comment:
Ideally these test.yml changes would be in their own commit, since they're not technically related to the auto build node selector change.

Reply:
made it in another commit